### PR TITLE
[Chongyang Bai] Add coco mAP evaluator and unit tests.

### PR DESCRIPTION
**Pull request by cy-bai**

### CODE REVIEW

**Comment by shonohs** [comment]
Could you update setup.py to include pycocotools as dependency?

**Comment by cy-bai** [comment]
> Could you update setup.py to include pycocotools as dependency?
 
done, thanks

**Comment by cy-bai** [comment]
> > Could you update setup.py to include pycocotools as dependency?
> 
> done, thanks

**Review by shonohs — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -14,7 +14,8 @@ This repo currently offers evaluation metrics for two vision tasks:
     - `EceLossEvaluator`: computes the [ECE loss](https://arxiv.org/pdf/1706.04599.pdf), i.e., the expected calibration error, given the model confidence and true labels for a set of data points. 
 - **Object detection**:
     - `MeanAveragePrecisionEvaluatorForSingleIOU`, `evaluators.MeanAveragePrecisionEvaluatorForMultipleIOUs`: computes the mean average precision (mAP), i.e. mean average precision across different classes, under single or multiple [IoU(s)](https://en.wikipedia.org/wiki/Jaccard_index).
-    
+    - `evaluators.CocoMeanAveragePrecisionEvaluator`: Coco mAP computation for multiple IoUs.
```
Comment (pj-ms) [inline README.md:17]: - Can we add it to vision_evaluation/__init__.py? 
- We can probably remove the MeanAveragePrecisionEvaluatorForSingleIOU and MeanAveragePrecisionEvaluatorForMultipleIOUs from the readme to demote the usage, or move them under CocoMeanAveragePrecisionEvaluator and update the readme to mark it as legacy.
- No need for "evaluators." prefix here. (actually same for MeanAveragePrecisionEvaluatorForMultipleIOUs, not part of this PR)

**Pull request merged**